testlib/src/main/resources/indent/IndentedWithTab.test

```diff
@@ -21,7 +26,10 @@ public class Java8Test {
 				throw new Exception();
 		}
 	}
-	
+
+	/**
+	 * Test enum.
+	 */
```

Comment:
```java
/** Test enum
* with weirdly formatted javadoc
* which indentstep should not change
*/ 
```

For example, let's say I hava `intro.txt` on which I am using `indentWithTabs`

```
**********
* INTRO *
**********
```

The exception where we're letting `indentStep` be "smart" is very narrow.